README.md

```diff
@@ -4,9 +4,10 @@ This repository contains Docker Compose configuration to run the full Inferno te
 
 ## Quick Start
 
-1. See [Inferno section](#Inferno) below for instructions on building
-2. See [Bulk Data Proxy section](#FHIR%20Bulk%20Data%20Proxy) below for instructions on building
-3. Run from the root directory of this repository:
+1. Install [Docker](https://www.docker.com/) for the host platform as well as the [docker-compose](https://docs.docker.com/compose/install/) tool (which may be included in the distribution, as is the case for Windows and MacOS).
+2. See [Inferno section](#Inferno) below for instructions on building
```

Comment:
Technically I don't think we need instructions to build an Inferno docker image since we have it on dockerhub now. So the user shouldn't need to build anything in order to run our inferno. We get it for free in the compose.